apps/onlyoffice.sh

```diff
@@ -48,6 +48,18 @@ sudo bash onlyoffice.sh"
     exit 1
 fi
 
+# check if apache2 evasive-mod is enabled and disable it because of compatibility issues
+if [ "$(Apache2ctl -M | grep evasive)" != "" ]
+then
+    msg_box "Please note: if you continue, the ddos protection for webservices will get disabled now, because of compatibility issues with onlyoffice."
+    if [[ "no" == $(ask_yes_or_no "So do you want to continue?")  ]]
+        then
```

Comment:
Skip the "So" here, I noticed you add that on all questions. 

It's enough with just "Do you want to continue?" :)

Reply:
I will fix it later, let it be, there are more changes to be made.

Reply:
you have already done this 👍

Reply:
It "will" not be disabled, the user can "choose" to disable it. There's a big difference.

Please change wording.